example/throttling-prevention-example/functions.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: pkg.crossplane.io/v1beta1
+kind: Function
+metadata:
+  name: function-azresourcegraph
+spec:
+  package: xpkg.upbound.io/upbound/function-azresourcegraph:v0.8.0
```

Comment:
This will need to be updated for the release of the function.

Reply:
this one will be bumped by renovate post release 👍